core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/ExternalFormWrapper.tsx

```diff
@@ -67,19 +68,22 @@ const preparePayload = ({
 };
 
 /**
- * Date pubValues need to be transformed to a Date type to pass validation
+ * Set all default values
+ * Special case: date pubValues need to be transformed to a Date type to pass validation
```

Comment:
Before, `defaultValues` would be based on the existing pubId. Now that we create, this was being set to `{}` which is _okay_ but not great, since that gives errors like switching from uncontrolled to controlled states when going from `undefined` to a `string` for many fields. This sets (hopefully) reasonably typed defaults for each type of schema field

Reply:
thanks for the explanation! i had seen those errors and wasn't sure where they were coming from!